src/Chirp.Web/Pages/TimelineModel.cshtml.cs

```diff
@@ -35,7 +41,7 @@ public ActionResult OnGet(string author, [FromQuery] int page = 1)
             page = 1;
         }
 
-        if (page < 1 || page > maxPage)
+        if ((page < 1 || page > maxPage) && _cheepRepository.QueryCheepCount(author) != 0)
```

Comment:
Why??

Reply:
Prevents infinite redirects in case there are no cheeps on the page.